src/__tests__/role.js

```diff
@@ -114,10 +132,11 @@ There are no accessible roles. But there might be some inaccessible roles. If yo
 `)
 })
 
-test('by default excludes elements which have visibility hidden', () => {
+ test('can exclude excludes elements which have visibility hidden', () => {
```

Comment:
Looks like I messed up the suggestion. my bad! 🙌 

```suggestion
test('can exclude elements which have visibility hidden', () => {
```